File: scratch/check_braces.js
const fs = require('fs');
const path = 'src/app/dashboard/submissions/[id]/page.tsx';
const content = fs.readFileSync(path, 'utf8');

const openBraces = (content.match(/\{/g) || []).length;
const closeBraces = (content.match(/\}/g) || []).length;
const openParens = (content.match(/\(/g) || []).length;
const closeParens = (content.match(/\)/g) || []).length;

console.log(`Braces: { : ${openBraces}, } : ${closeBraces}, Diff: ${openBraces - closeBraces}`);
console.log(`Parens: ( : ${openParens}, ) : ${closeParens}, Diff: ${openParens - closeParens}`);

File: scratch/check_braces.py
import os

path = r'c:\Users\Dell\Downloads\cohort\src\app\dashboard\submissions\[id]\page.tsx'
with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

open_braces = content.count('{')
close_braces = content.count('}')
open_parens = content.count('(')
close_parens = content.count(')')

print(f"Braces: {{: {open_braces}, }}: {close_braces}, Diff: {open_braces - close_braces}")
print(f"Parens: (: {open_parens}, ): {close_parens}, Diff: {open_parens - close_parens}")
